es.js

```diff
@@ -751,7 +751,7 @@ server.config.bDrop Caída de bala
 server.config.thirdPerson Tercera persona
 server.config.requireCaptcha Requiere captcha
 server.config.nameTags Ocultar nombres
-server.config.nameTagsFR Hide Nametags (Allies)
+server.config.nameTagsFR Ocultar nombres (Alianzas)
```

Comment:
```suggestion
server.config.nameTagsFR Ocultar nombres (Aliados)
```
The best translation for "Allies" would actually be "Aliados" because "Alianzas" means "Alliances"

Reply:
Agree with this one, for sure

Buen Trabajo!